src/components/SidePanel/NavGroup/BaseNavGroup/index.tsx

```diff
@@ -25,6 +26,7 @@ const BaseNavGroup = <T extends {}>({
   items,
   navGroupTitle,
   defaultMessage,
+  additionalHeaderText,
```

Comment:
I feel like this prop is misleading since its a button that opens a modal but I can't think of an alternative name. 😅

Reply:
Technically you could pass any kind of element here, since the modal logic is all external and not tied to this component